v21.1/topology-follow-the-workload.md

```diff
@@ -13,6 +13,8 @@ In a multi-region deployment, follow-the-workload is the default behavior for ta
 
 {{site.data.alerts.callout_success}}
 If read performance is your main focus for a table, but you want low-latency reads everywhere instead of just in the most active region, consider [Global Tables](global-tables.html) or [Follower Reads](topology-follower-reads.html).
+
+Note that if you start using the [multi-region SQL abstractions](multiregion-overview.html), CockroachDB will no longer provide the follow-the-workload behavior described on this page.
```

Comment:
Consider scoping this to a single database. So "if you start using the [multi-region SQL abstractions](multiregion-overview.html) for a database" ... "described on this page for that database."

Reply:
Done!